PORT-2479 Plenty of docs fixes and updates (#151)
port-labs/port-docs: docs/software-catalog/entity/tutorial.md

properties': {"version": "2.29"}
```

### Update Entity identifier

> In order to change the identifier of an Entity you can use one of the following options:

Reviewer: ```suggestion
There are two methods to to change an Entity's identifier:
```

#### From the UI

1. In either the matching Blueprint page or [Entity page](./entity.md#entity-page), click the 3 dots (`...`) button and then click the `Edit` button;
2. Update the value of the `identifier` key to the new identifier;
3. Click `Update`.

#### PUT request

> Make an HTTP PUT request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{old_entity_identifier}` with the Entity definition JSON, after updating the value of the `identifier` key in the request body to the new identifier.

Reviewer: Same problem with the sentence as above.
